DDD.Functions/SubmitVote.cs

```diff
@@ -63,6 +65,16 @@ VotingConfig voting
                 return new StatusCodeResult((int) HttpStatusCode.BadRequest);
             }
 
+            // Get tickets
+            var ticketsRepo = await tickets.GetRepositoryAsync();
+            var allTickets = await ticketsRepo.GetAllAsync(conference.ConferenceInstance);
```

Comment:
We should add a new method to look up a ticket by partition and row key rather than loading all the tickets

Reply:
Used GetAsync instead of GetAllAsync